src/sections/dataset/Dataset.tsx

```diff
@@ -25,7 +26,7 @@ export function Dataset({ datasetRepository, id }: DatasetProps) {
           <Col sm={9}>Citation Block</Col>
         </Row>
         <Row>
-          <Col sm={9}>Summary Block</Col>
+          <DatasetSummary datasetRepository={datasetRepository} id={id}></DatasetSummary>
```

Comment:
Ok, I will remove datasetRepository here, but since the License inside the DatasetSummary needs dataset data, I think I need to pass dataset, instead of dataset.summaryFields, does this look ok?

`<DatasetSummary dataset={dataset} />`

Reply:
Or I could pass the two domain objects separately: 

`<DatasetSummary fields={dataset.summaryFields}, license={dataset.license} />`